<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionRedo_R"/>
    <addaction name="separator"/>
    <addaction name="actionCut_T"/>
    <addaction name="actionCopy_C"/>
    <addaction name="actionPaste_P"/>
    <addaction name="actionDelete_D"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newAction">
   <property name="text">
    <string>new(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>new(O)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>open(&amp;O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>save(&amp;S)</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>save as(&amp;A)</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>print(&amp;P)</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>quit(&amp;Q)</string>
   </property>
  </action>
  <action name="actionRedo_R">
   <property name="text">
    <string>redo(&amp;R)</string>
   </property>
  </action>
  <action name="actionCut_T">
   <property name="text">
    <string>cut(&amp;T)</string>
   </property>
  </action>
  <action name="actionCopy_C">
   <property name="text">
    <string>copy(&amp;C)</string>
   </property>
  </action>
  <action name="actionPaste_P">
   <property name="text">
    <string>paste(&amp;P)</string>
   </property>
  </action>
  <action name="actionDelete_D">
   <property name="text">
    <string>delete(&amp;D)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
